middleware/logger.go

```diff
@@ -43,34 +42,27 @@ func Logger(next chi.Handler) chi.Handler {
 	return chi.HandlerFunc(fn)
 }
 
-func printStart(reqID string, r *http.Request) {
+func requestPrefix(reqID string, r *http.Request) string {
 	var buf bytes.Buffer
 
+	cW(&buf, bMagenta, "%s ", r.RemoteAddr)
 	if reqID != "" {
 		cW(&buf, nYellow, "[%s] ", reqID)
 	}
-	buf.WriteString("Started ")
+	cW(&buf, nBlue, "\"")
 	cW(&buf, bMagenta, "%s ", r.Method)
-	cW(&buf, nBlue, "%q ", r.URL.String())
-	buf.WriteString("from ")
-	buf.WriteString(r.RemoteAddr)
+	cW(&buf, nBlue, fmt.Sprintf("%q ", r.URL.String()+" "+r.Proto)[1:])
```

Comment:
The `[1:]` is to skip the first `"` added by the `%q` formatter. It's a trick to be able to put the URI and the protocol within the same quotation marks, while we're giving the method a different color.